sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerClient.java

```diff
@@ -88,7 +90,7 @@
      * @param context Additional context that is passed through the HTTP pipeline during the service call.
      *
      * @return A {@link SyncPoller} to poll the progress of the recognize custom form operation until it has completed,
-     * has failed, or has been cancelled. The completed operation returns a List of {@link RecognizedForm}.
```

Comment:
NIT: line 78 says `Recognizes receipt data from documents` should be `Recognizes form data from documents`